Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
+
+# Set up environment
+ENV LANG C.UTF-8
+WORKDIR /srv
+
+# System dependencies
+RUN apt-get update && apt-get install --yes python3-pip
+
+# Set git commit ID
+ARG COMMIT_ID
+RUN test -n "${COMMIT_ID}"
+RUN echo "${COMMIT_ID}" > version-info.txt
```

Comment:
We no longer need to create the `version-info.txt` file if we use the `TALISKER_REVISION_ID` environment variable